### PR TITLE
Configs for more flexible deployment

### PR DESCRIPTION
**Pull request by petertrr**
* Gateway/actuator: Don't check password if subnet matches (checking request origin is enough of a security measure, and also authenticated health checks are not supported well enough in Kubernetes)
* Backend/postprocessor: Read secrets base path from env. A bit of un-hardcoding paths from env postprocessor to allow more flexible configuration in other environemtns

Fixme (that we kind of forgot about): logs from postprocessor are not visible, see https://stackoverflow.com/a/42840529

### CODE REVIEW

File: api-gateway/src/main/kotlin/org/cqfn/save/gateway/security/WebSecurityConfig.kt [hunk 1]
```diff
@@ -185,22 +183,8 @@ class WebSecurityConfig(
                     }
                 )
             )
-            .authenticated()
+            .permitAll()
     }
-        .and().httpBasic().run {
-            val userFromProps = configurationProperties.basicCredentials?.split(' ')?.run {
```
Comment (kgevorkyan) [inline api-gateway/src/main/kotlin/org/cqfn/save/gateway/security/WebSecurityConfig.kt:191]: do we still need corresponding field `basicCredentials` in `ConfigurationProperties`?

File: api-gateway/src/main/kotlin/org/cqfn/save/gateway/security/WebSecurityConfig.kt [hunk 2]
```diff
@@ -185,22 +183,8 @@ class WebSecurityConfig(
                     }
                 )
             )
-            .authenticated()
+            .permitAll()
```
Comment (kgevorkyan) [inline api-gateway/src/main/kotlin/org/cqfn/save/gateway/security/WebSecurityConfig.kt:186]: Maybe add some comments here too, why do we `permitAll` from now

File: api-gateway/src/main/kotlin/org/cqfn/save/gateway/security/WebSecurityConfig.kt (hunk 1 above)
Comment (petertrr) [reply api-gateway/src/main/kotlin/org/cqfn/save/gateway/security/WebSecurityConfig.kt:191]: No, removed

File: api-gateway/src/main/kotlin/org/cqfn/save/gateway/security/WebSecurityConfig.kt (hunk 2 above)
Comment (petertrr) [reply api-gateway/src/main/kotlin/org/cqfn/save/gateway/security/WebSecurityConfig.kt:187]: Yes, added

**Pull request merged**